src/Viewer/services/decoder/FileManager.js

```diff
@@ -303,6 +165,7 @@ class FileManager {
         try {
             while (this._irStreamReader.indexNextLogEvent(this._logEventOffsets)) {}
         } catch (error) {
+            console.log(error);
```

Comment:
Wouldn't this be a duplicate error message since stream truncation has an error message and the thrown exception generates an error message?